### DISCUSSION TITLE
Book: 404 page which automatically includes book navbar etc.

### DISCUSSION THREAD

**Discussion by meierluk** [quarto-dev/quarto-cli#9988] [Q&A] [answered]
### Description

It would be nice if a custom 404.qmd would automatically inherit all the properties of the corresponding book project, i.e., render with the navbar etc. (as seems to have been the case with bookdown).

**Comment by mcanouil** [comment]
It's already the case at least for website.

| HTML | SOURCE |
|--------|--------|
| <img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/9924a50d-ec93-476f-9912-f5959639e9b1"> | <img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5f71529e-3b66-4e76-a3e5-10dc6e6d8589"> | 


I would have believe the same applies for Book projects.
Do you have an example in which it does not work?

**Comment by meierluk** [comment]
If I use the (only-slightly edited) book template, I get this [404.html](https://meierluk.github.io/quarto-test/404.html), compare with the [standard book layout](https://meierluk.github.io/quarto-test/).

The source is available on [github.com/meierluk/quarto-test/](https://github.com/meierluk/quarto-test/).

**Reply by mcanouil** [reply]
Thanks for the repository.

Side note: be aware that using "here" as the text make your content not accessible as "here" is not meaningful, *i.e.*, we have no clue of what "here" is about or link to. See [W3.org tips for writing accessible content such as making link text meaningful](https://www.w3.org/WAI/tips/writing/#make-link-text-meaningful).

**Reply by meierluk** [reply]
Good point! I have updated it accordingly.

**Comment by meierluk** [comment]
According to the [documentation](https://quarto.org/docs/books/) "HTML books are actually just a special type of Quarto Website". Should it then considered to be an issue?

**Reply by mcanouil** [reply]
Possibly, but we haven't looked at your example just yet.
If there is an issue, we'll open a GitHub Issue to track the bug.

**Reply by cderv** [reply]
Yes this is an issue I can reproduce. I'll move as a bug, just need to check more. Because I think this is related to our other sidebar

- https://github.com/quarto-dev/quarto-cli/issues/9242 
- https://github.com/quarto-dev/quarto-cli/issues/9728
which were supposed to be solved by 
- https://github.com/quarto-dev/quarto-cli/pull/9741

but it was reverted because it broke some stuff. 

So complicated stories.

**Reply by cderv** [reply]
So I can confirm that this is same issue as the one I linked and that commit which breaks this is ff820aa32f00f0a5cc3f59598c660da945f6707c with the change regarding sidebar 
* https://github.com/quarto-dev/quarto-cli/issues/5689

I'll open and link to others

**Reply by cderv** [reply, marked as answer]
Tracking in 
- https://github.com/quarto-dev/quarto-cli/issues/10014
